gpt_index/indices/composability/graph.py

```diff
@@ -100,6 +100,9 @@ def from_indices(
             index_node = IndexNode(
                 text=summary,
                 index_id=index.index_struct.index_id,
+                relationships={
```

Comment:
why is this necessary?

Reply:
because in vector_store/base.py there's these lines

```
doc_id = id_to_node_map[id].ref_doc_id
if doc_id is None:
   raise ValueError("Reference doc id is None.")
```

Basically the ref_doc_id must be defined for every node that we insert into a vector store. in a composed index setting, we create an index node and figured ref_doc_id should point to the index id